docs/development/i18n.rst

```diff
@@ -64,6 +59,19 @@ files into the corresponding ``*.j2`` file and committed to the
 SecureDrop repository. They are then installed when configuring Tails
 with the ``tasks/create_desktop_shortcuts.yml`` tasks.
 
+We don't expect translators to deal with all these files directly, of
+course. Translation happens on our `Weblate`_ server, which is
+configured to use a fork of the main `SecureDrop git repository`_.
+
+At the start of the the release process, the localization manager
+collects string changes on the ``develop`` branch and merges them to
+the ``i18n`` branch of the ``securedrop-i18n`` repository. The changes
+will then appear in Weblate, and translation can begin. Once complete,
+the localization manager collects the changes to the PO files on
+``securedrop-i18n/i18n`` in a pull request for ``develop``. Once that
+pull request is merged, the translations are backported to the release
+branch.
```

Comment:
this happens the day before the release, correct? a comment about when to backport would be useful.